common/traits/wc_genetic_traits.txt

```diff
@@ -224,4 +225,49 @@ magic_good_noninheritable = {
 			desc = trait_magic_good_noninheritable_character_desc
 		}
 	}
-}
\ No newline at end of file
+}
+
+### Other
+dire = {
```

Comment:
What about giving them the same opinion bonuses the Giant trait has?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/126666116-d030e708-e36c-404d-863d-30c91b8b8b79.png)

</details>

Reply:
Added in d88812f4c9f9999269af6c1029068c16fbfb586d.